### PR TITLE
Feature/enable simple sign in denmark

### PR DESCRIPTION
**Pull request by FredrikAreschoug**
## What?
- Enable simple sign in denmark
- Clean up of redirect

## Why?
- We want to use simple sign in denmark

## Optional checklist
- [x] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/main/kotlin/com/hedvig/underwriter/service/quotesSignDataStrategies/SignStrategyService.kt [hunk 1]
```diff
@@ -28,7 +27,7 @@ class SignStrategyService(
             return StartSignErrors.noQuotes
         }
 
-        if (strategy.size > 1) {
+        if (strategy.size > 1 || quotes.markets.size > 1) {
```
Comment (vonElfvin) [inline src/main/kotlin/com/hedvig/underwriter/service/quotesSignDataStrategies/SignStrategyService.kt:30]: Can't we just add a new StartSignError?

File: src/main/kotlin/com/hedvig/underwriter/service/quotesSignDataStrategies/RedirectSignStrategy.kt [hunk 2]
```diff
@@ -1,88 +0,0 @@
...
-import com.hedvig.underwriter.serviceIntegration.memberService.dtos.NationalIdentification
-import com.hedvig.underwriter.serviceIntegration.memberService.dtos.RedirectCountry
-import com.hedvig.underwriter.serviceIntegration.memberService.dtos.UnderwriterStartSignSessionResponse
-import com.hedvig.underwriter.serviceIntegration.memberService.dtos.Nationality
-import org.springframework.stereotype.Service
-import java.util.UUID
-import kotlin.IllegalStateException
-
-@Service
-class RedirectSignStrategy(
```
Comment (vonElfvin) [inline src/main/kotlin/com/hedvig/underwriter/service/quotesSignDataStrategies/RedirectSignStrategy.kt:19]: Would we need this if we went to a new market anytime soon?

**Review by vonElfvin — APPROVED** [review]
Looks good, added two questions that are maybe worth considering

File: src/main/kotlin/com/hedvig/underwriter/service/quotesSignDataStrategies/RedirectSignStrategy.kt (hunk 2 above)
Comment (FredrikAreschoug) [reply src/main/kotlin/com/hedvig/underwriter/service/quotesSignDataStrategies/RedirectSignStrategy.kt:19]: I don't know but you can always find it in the git history

File: src/main/kotlin/com/hedvig/underwriter/service/quotesSignDataStrategies/SignStrategyService.kt (hunk 1 above)
Comment (FredrikAreschoug) [reply src/main/kotlin/com/hedvig/underwriter/service/quotesSignDataStrategies/SignStrategyService.kt:30]: Yes we can but if I don't see why the thing is that the quotes can not be bundled if they are of different markets

**Pull request merged**